bootstrap.d/dev-libs.yml

```diff
@@ -242,6 +242,10 @@ packages:
       - args: ['ninja', 'install']
         environ:
           DESTDIR: '@THIS_COLLECT_DIR@'
+    scripts:
+        post_install:
+          - args: 'glib-compile-schemas /usr/share/glib-2.0/schemas'
+          - args: 'gio-querymodules /usr/lib/gio/modules'
```

Comment:
Why is this a shell invocation and not just a argument array (same for the other instances)? AFAICS the previous patch didn't work because it was missing a comma in the arg array. But `['glib-compile-schemas', '/usr/share/glib-2.0/schemas']` should work, right?

Reply:
You might be on to something there. I’ll change it around and test it. If that works, I’ll use that over shell invocations.